db/people.yml

```diff
@@ -2040,3 +2040,263 @@ zzgong:
   service: []
   skills: []
   title: Dr.
+rdifrancesco:
+  active: false
+  aka:
+    - Remus DiFrancesco
+  avatar: https://static-exp1.licdn.com/sc/h/djzv59yelk5urv2ujlazfyvrk
+  bio: bio
+  education:
+    - begin_year: 1992
+      degree: PhD
+      institution: msu
+      end_year: 1999
+      end_month: 12
+      group: bg
+      advisor: sbillinge
+  employment:
+    - begin_year: 2006
+      begin_month: 01
+      position: Senior Support Consultant
+      organization: SAP
+    - begin_year: 1992
+      position: Research Associate
```

Comment:
These should **all** be ``Research Assistant`` not ``Research Associate``, which is a post-doc title